text/0097-log-data-model.md

```diff
@@ -0,0 +1,972 @@
+# Log Data Model
+
+Introduce Data Model for Log Records as it is understood by OpenTelemetry.
+
+* [Motivation](#motivation)
+* [Design Notes](#design-notes)
+  * [Requirements](#requirements)
+  * [Field Kinds](#field-kinds)
+* [Log and Event Record Definition](#log-and-event-record-definition)
+  * [Field: `Timestamp`](#field-timestamp)
+  * [Trace Context Fields](#trace-context-fields)
+    * [Field: `TraceId`](#field-traceid)
+    * [Field: `SpanId`](#field-spanid)
+    * [Field: `TraceFlags`](#field-traceflags)
+  * [Severity Fields](#severity-fields)
+    * [Field: `SeverityText`](#field-severitytext)
+    * [Field: `SeverityNumber`](#field-severitynumber)
+    * [Displaying Severity](#displaying-severity)
+    * [Comparing Severity](#comparing-severity)
+  * [Field: `ShortName`](#field-shortname)
+  * [Field: `Body`](#field-body)
+  * [Field: `Resource`](#field-resource)
+  * [Field: `Attributes`](#field-attributes)
+* [Example Log Records](#example-log-records)
+* [Open Questions](#open-questions)
+* [Alternate Design](#alternate-design)
+* [Prior Art](#prior-art)
+  * [RFC5424 Syslog](#rfc5424-syslog)
+  * [Fluentd Forward Protocol Model](#fluentd-forward-protocol-model)
+* [Appendix A. Example Mappings.](#appendix-a-example-mappings)
+  * [RFC5424 Syslog](#rfc5424-syslog-1)
+  * [Windows Event Log](#windows-event-log)
+  * [SignalFx Events](#signalfx-events)
+  * [Splunk HEC](#splunk-hec)
+  * [Log4j](#log4j)
+  * [Zap](#zap)
+  * [Apache](#apache)
+  * [CloudTrail Log Event](#cloudtrail-log-event)
+* [Appendix B: `SeverityNumber` example mappings.](#appendix-b-severitynumber-example-mappings)
+* [References](#references)
+
+## Motivation
+
+This is a proposal of a data model and semantic conventions that allow to
+represent logs from various sources: application log files, machine generated
+events, system logs, etc. Existing log formats can be unambiguously mapped to
```

Comment:
nit: are terms "event", "log", "log record", and "record" used interchangeably in this document? Perhaps we can clarify that event is a log event in this document, not a span's event.

Reply:
This OTEP uses the terms "Log" and "Log Record" defined in the vocabulary: https://github.com/open-telemetry/oteps/blob/master/text/logs/0091-logs-vocabulary.md
The vocabulary refers to events but does not formally define the term "event", we can add it for clarity.

I added a clarification to this OTEP that it is about Standalone logs.

Reply:
I have a blog post that attempts to put together a taxonomy of logging, since I haven't seen this elsewhere https://tersesystems.com/blog/2020/03/10/a-taxonomy-of-logging/

Reply:
Thanks for the link. Resolving this comment. Feel free to reopen if needed.